starboard/build/config/base_configuration.gni

```diff
@@ -197,3 +197,9 @@ if (current_toolchain == starboard_toolchain &&
     shared_library_configs = []
   }
 }
+
+# Add the POSIX-based wrapper linker flags here. The referenced
+# configuration file includes the necessary conditions to decide when
+# to include the flags.
+shared_library_configs +=
```

Comment:
If possible, let's just add this config to the `//starboard/build/config/modular` config (if it also works fine when applied to executables).